modules/mvt/src/lib/binary-vector-tile/features-to-binary.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable indent */
```

Comment:
Should we just make this a global rule in `.eslintrc.js`?

Reply:
I couldn't. It didn't work globally for some reason